docs/code-search/code-navigation/precise_code_navigation.mdx

```diff
@@ -0,0 +1,80 @@
+# Precise Code Navigation
+
+<p className="subtitle">Learn and understand about Precise Code Navigation.</p>
+
+Precise Code Navigation is an opt-in feature that is enabled from your admin settings and requires you to upload indexes for each repository to your Sourcegraph instance. Once setup is complete on Sourcegraph, precise code navigation is available for use across popular development tools:
+
+- On the Sourcegraph web UI
+- On code files on your code host, via [integrations](/integration/)
+- On diffs in your code review tool, via integrations
+- Via the [Sourcegraph API](/api/graphql)
+
+Sourcegraph automatically uses Precise Code Navigation whenever available, and Search-based Code Navigation is used as a fallback when precise navigation is not available.
+
+Precise code navigation relies on the following technologies:
+
+- [SCIP](https://github.com/sourcegraph/scip) (SCIP Code Intelligence Protocol) and
+- [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification)
+(Language Server Index Format) data to deliver precomputed code navigation. It provides fast and highly accurate code navigation but needs to be periodically generated and uploaded to your Sourcegraph instance. Precise code navigation is an opt-in feature: repositories for which you have not uploaded indexes will continue to use the search-based code navigation.
```

Comment:
```suggestion
Precise code navigation relies on the open source [SCIP Code Intelligence Protocol](https://github.com/sourcegraph/scip), which is a language-agnostic protocol for indexing source code.
```

We can remove the reference to LSIF as we no longer support LSIF directly, we only support it via conversion to SCIP.